portfolio/src/main/webapp/index.html

```diff
@@ -543,5 +543,7 @@ <h2 class="text-uppercase">James Scurlock</h2>
     <script src="assets/mail/contact_me.js"></script>
     <!-- Core theme JS-->
     <script src="js/scripts.js"></script>
+    <!-- Custom JS-->
+    <script src="js/custom-script.js"></script>
```

Comment:
Please choose a more descriptive file name.

Reply:
May you please give an example? The script is intended to separate our code from what we got off bootstrap.

Reply:
Line 543 is a very good example as the "contact_me.js" reflects the intention. But "custom-script.js" is too generic.

Reply:
I messaged you so we could discuss this.